docs/src/main/asciidoc/security-jdbc.adoc

```diff
@@ -184,7 +184,7 @@ INSERT INTO test_user (id, username, password, role) VALUES (2, 'user','user', '
 [NOTE]
 ====
 It is probably useless, but we kindly remind you that you must not store clear-text passwords in production environment ;-).
-The `elytron-security-jdbc` offers a built-in bcrypt password mapper.
+The `elytron-security-jdbc` offers a built-in bcrypt password mapper. You can see the section `Define the user entity` for more details at https://quarkus.io/guides/security-getting-started-tutorial#define-the-user-entity.
```

Comment:
```suggestion
The `elytron-security-jdbc` offers a built-in bcrypt password mapper. You can see the section `Define the user entity` for more details at xref:security-getting-started-tutorial.adoc#define-the-user-entity.
```

You also need to add `[[define-the-user-entity]]` to the 

https://github.com/quarkusio/quarkus/blob/e1c4c96ee8a07149102c830999cbea1edadcf4d2/docs/src/main/asciidoc/security-getting-started-tutorial.adoc?plain=1#L231